edenai_apis/apis/lovoai/lovoai_api.py

```
@@ -289,3 +294,187 @@ def audio__text_to_speech_async__get_job_result(
+        # Apply speed (LovoAI supports 0.5-1.5)
+        resolved_speed = normalize_speed_for_lovoai(speed)
+
+        payload = json.dumps(
+            {
+                "text": text,
+                "speaker": speaker_id,
+                "speed": resolved_speed,
+            }
+        )
+
+        response = requests.post(
```

Ruff (0.14.13)

Probable use of `requests` call without timeout

(S113)